### PR TITLE
Disable link shares of disabled users

### LINKED ISSUE #10869: Disabled users should have their shared file links disabled?
If a user shares files and generates links those should be disabled once the account is marked "Disabled"? Currently they still work.

### PR DESCRIPTION
**Pull request by rullzer**
Fixes #10869

Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Comment by skjnldsv** [comment]
What is the status here? :)

File: .htaccess [hunk 1]
```diff
@@ -67,3 +67,7 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
```
Comment (nickvergessen) [inline .htaccess]: woopsy

**Review by nickvergessen — APPROVED** [review]

**Comment by skjnldsv** [comment]
Backports ?

**Review by skjnldsv — APPROVED** [review]

**Comment by rullzer** [comment]
> Backports ?

Only if they work clean. Has been the case since ages so...

**Pull request merged**